src/components/chatbox/message.module.css

```diff
@@ -0,0 +1,94 @@
+.message {
+	display: flex;
+	flex-direction: row;
+	gap: 12px;
+	padding: 20px 16px;
+	background: var(--token-color-palette-neutral-0);
+}
+
+.message_user {
```

Comment:
We've typically use camel case for classnames, I'd prefer to maintain this consistency unless there's a good reason to deviate.

Reply:
Camel cased! c398f888f